trisacrypto/directory
# sc-5170 gds apis sentry integration

### Scope of changes

Integrate Sentry with the GDS API, Admin API, and Members API

### Type of change


### Acceptance criteria

To test this change:
1. Set required environment variables for each component; the sentry DSN values for each can be found on the settings page in the sentry dashboard
2. Trigger calls via the gds CLI and verify panics and perf tracing shows up in the sentry dashboard

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [x]  I have updated the dependencies list
- [x]  I have recompiled and included new protocol buffers to reflect changes I made
- [x]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
